src/main/java/thePackmaster/patches/contentcreatorpack/VoyagedPatch.java

```diff
@@ -0,0 +1,19 @@
+package thePackmaster.patches.contentcreatorpack;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import thePackmaster.actions.contentcreatorpack.EnableDrawnDuringTurnAction;
+
+public class VoyagedPatch {
+
+    @SpirePatch(
+            clz = AbstractPlayer.class,
+            method = "applyStartOfTurnPostDrawRelics"
+    )
+    public static class AbstractPlayerApplyStartOfTurnPostDrawRelicsPatch {
+        public static void Prefix(AbstractPlayer __instance) {
+            AbstractDungeon.actionManager.addToBottom(new EnableDrawnDuringTurnAction());
```

Comment:
It looks like this patch and PreDrawPatch are doing the same thing, but in slightly different ways? This is the way (adding an action after one of the post draw methods) that I think works better and described in my comment on PreDrawPatch, so maybe just delete the current PreDrawPatch entirely and rely on this patch? (Maybe renamed to not use the adventurer-specific Voyaged term.)

Let me know if I'm missing something.

Reply:
Yeah this also looks suspicious, i'll try with your suggestion and see what happens.

Reply:
Actually i'll let @DarkVexon take care of this because I feel this is probably some weird edge case that I won't find in a test.

Reply:
The three patches work together. One disables counting cards at the start of the turn, the other adds the action that re-enables it, and the OnDrawCard/TrackCardsDrawnDuringTurnPatch is what triggers on draw, checks against the variable and actually counts the drawn card. I've renamed the classes so hopefully it's a little more clear.

Reply:
Thanks. I understand what you're saying -- having the code spread out across multiple patches made it hard to put everything together. It would be a lot more readable if all the patches were consolidated into one place (in addition to the naming changes you mentioned).